src/components/LoginForm/LoginForm.tsx

```diff
@@ -6,36 +6,52 @@ import "./LoginForm.scss"
 import staticText from "../../languages/en.pl";
 import {Input} from "../Formik/Input/Input";
 import {SubmitBtn} from "../common/SubmitBtn/SubmitBtn";
+import {useLoginMutation} from "../../api/authApiSlice";
+import {useDispatch} from "react-redux";
+import {setCredentials} from "../../store/auth/authSlice";
+import {navigateToDefaultRoute} from "../../utils/navigation/navigation";
+import {useNavigate} from "react-router";
 
 interface LoginValues {
     login:string,
     password:string,
 }
 
-export const LoginForm = () => (
-    <>
+export const LoginForm = () => {
+    const [login, {isLoading, isError}] = useLoginMutation();
+    const dispatch = useDispatch();
+    const navigator = useNavigate();
+
+    return <>
         <Formik
             initialValues={{
                 login: "",
                 password: "",
             }
             }
             validationSchema={Yup.object({
-                login:Yup.string()
+                login: Yup.string()
                     .required("Pole wymagane")
-                    .min(4,"Login musi mieć minimum 4 znaki"),
-                password:Yup.string()
+                    .min(4, "Login musi mieć minimum 4 znaki"),
```

Comment:
teksty statyczne umieściłbym w pliku en.pl.js w folderze language

Reply:
Tak, ale to jest z branchu kto implementował formika